### ISSUE TITLE
Fails to link under MS-DOS 

### ISSUE THREAD

**Issue by dmitrygerasimuk** [cml37/dos-push-notifications-tsr#3]
I failed to wlink it on a clean system, it fails with 
wlink: Argument list is too big
Can you provide more info how do you build it or maybe just release an exe

**Comment by cml37** [comment]
@dmitrygerasimuk can you confirm you are using Open Watcom 1.9 and not a newer version?

**Comment by dmitrygerasimuk** [comment]
> @dmitrygerasimuk can you confirm you are using Open Watcom 1.9 and not a newer version?

Yes, my version is 1.9. I think the issue is that I'm building it under DOS, I guess there should be a workaround to get all the args from file, but I don't know the Watcom syntax for that

**Comment by cml37** [comment]
@dmitrygerasimuk got it.  Ya, I am building under Windows 10.  I can build and send you an executable.  You can send me an email from the about page [here](https://www.youtube.com/c/RetroTechChris/about) if you'd like and I will send you an executable!
